api/src/util/config.ts

```diff
@@ -23,6 +23,7 @@ const BUCKETEER_AWS_REGION = process.env.BUCKETEER_AWS_REGION || 'us-east-1';
 const BUCKETEER_BUCKET_NAME = process.env.BUCKETEER_BUCKET_NAME || 'documents';
 const BUCKETEER_ENDPOINT = process.env.BUCKETEER_ENDPOINT || '';
 const USE_SSL = process.env.USE_SSL || false;
+const MAX_JSON_SIZE = process.env.MAX_JSON_SIZE || '100kb';
```

Comment:
I think we need to support more than 100kb. I do not think that is big enough. People will get frustrated if there resume is too big and they can't submit it. I would say we should support on the order of MBs

Reply:
Sounds good, bumping up the default values